config/locales/en.yml

```diff
@@ -2085,6 +2091,18 @@ en:
   if_matches_come_from_a_comprehensive_list:  If matches come from a comprehensive list a green bar will show here.
   if_this_is_a_scientific_name: If this is a scientific name, is it currently accepted?
   if_this_link_will_apply_to_descendant: If this link will apply to descendent taxa, you can use template tags to fill in the taxon name in the URL
+  if_you_are_not_uploading: |
+    If you are not uploading an IUCN RedList range map, this should be because:
+    <ul>
+      <li><p>
+        <b>IUCN RedList range map for this taxon is unsutable:</b> for example, you’ve manually narrowed
+        it to reflect a taxonomic split not represented in the IUCN RedList, or the IUCN
+        RedList range map has major errors making its display problematic.</p></li>
+      <li><p>
+        <b>Taxon is not in the IUCN RedList:</b> for example, the taxon is lumped with another taxon in the IUCN
+        RedList so you’ve manually carved it off here, or if the taxon lacks an IUCN RedList assessment
+        and range map.</p></li>
+    </ul>
```

Comment:
Again, I would avoid embedding HTML in the translatable text because any changes in markup will require re-translation, and translators tend to make mistakes with the markup. I think it also helps to name view-specific keys like this with some consistent prefix so they stay together in the source file, so like

```yaml
taxon_range_guidelines_if_not_uploading: "If you are not uploading an IUCN RedList range map, this should be because:"
taxon_range_guidelines_iucn_unsuitable: IUCN RedList range map for this taxon is unsutable
```

etc. Also, FWIW, have a translation for bold text followed by a colon: `bold_label_colon_value_html`.